server/sheet.go

```diff
@@ -67,6 +72,194 @@ func (s *Server) registerSheetRoutes(g *echo.Group) {
 		return nil
 	})
 
+	g.POST("/project/:projectID/syncsheet", func(c echo.Context) error {
```

Comment:
Since we are here, let's change this to sync-sheet per https://github.com/bytebase/bytebase/pull/1002